array_test.go

```diff
@@ -874,6 +894,10 @@ func TestArray_ConsistsOf(t *testing.T) {
 		value.NotConsistsOf(123, "foo")
 		value.chain.assertFailed(t)
 		value.chain.clearFailed()
+
+		value.NotConsistsOf(make(chan int)) // bad type make(chan int)
+		value.chain.assertFailed(t)
+		value.chain.clearFailed()
```

Comment:
Isn't this the same as:

```go
		value.NotConsistsOf(func() {})
		value.chain.assertFailed(t)
		value.chain.clearFailed()
```

which we already have?

Reply:
you're right, it follows the same path in the function, so i've removed it